### PR TITLE
Update dependencies to match 248 Core.

### PR DESCRIPTION
**Pull request by sfdctaka**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,15 +14,15 @@
     "prettier:verify": "prettier --list-different \"force-app/**/*.js\""
   },
   "devDependencies": {
-    "@lwc/engine-server": "^2.48.0",
-    "@lwc/eslint-plugin-lwc": "^1.6.3",
+    "@lwc/engine-server": "^3.8.0",
```
Comment (sfdctaka) [inline package.json:17]: ver 3.x is used for 248(Winter 24) according to https://lwc.dev/guide/versioning. Using v3.8.0.

image

File: package.json [hunk 2]
```diff
@@ -14,15 +14,15 @@
...
+    "@lwc/eslint-plugin-lwc": "^1.7.2",
     "@prettier/plugin-xml": "^3.2.0",
     "@sa11y/jest": "^5.2.0",
     "@salesforce/eslint-config-lwc": "^3.5.2",
     "@salesforce/eslint-plugin-aura": "^2.1.0",
     "@salesforce/eslint-plugin-lightning": "^1.0.0",
-    "@salesforce/eslint-plugin-lwc-graph-analyzer": "^0.7.1",
-    "@salesforce/sfdx-lwc-jest": "^2.0.1",
+    "@salesforce/eslint-plugin-lwc-graph-analyzer": "^0.9.0",
```
Comment (sfdctaka) [inline package.json:24]: Latest graph analyzer uses Komaci static analyzer that is updated for 248.

File: package.json [hunk 3]
```diff
@@ -14,15 +14,15 @@
...
+    "@salesforce/sfdx-lwc-jest": "^3.1.1",
```
Comment (sfdctaka) [inline package.json:25]: Updating `sfdx-lwc-jest` got rid of all 'could not resolve dependency' warnings. Instead, now, have deprecation warnings when running `npm install`.

image

**Review by maliroteh-sf — APPROVED** [review]
👍

**Pull request merged**